public/components/chat_welcome_message.tsx

```diff
@@ -19,7 +20,7 @@ export const WelcomeMessage = (props: Props) => {
       titleSize="s"
       body={
         <EuiText color="default">
-          <p>Welcome {props.username} to the OpenSearch Assistant!</p>
+          <p>Welcome{usernamePlaceHolder}to the OpenSearch Assistant!</p>
```

Comment:
Shall we add empty space here? `Welcometo` would be weird.

Reply:
Space is predefined in `usernamePlaceHolder`.

Reply:
For username case, there is one space before and after the name, and only one space in the case without username.

Reply:
Space for non username case has been added, you can check it again.